spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AppRegistryController.java

```diff
@@ -252,6 +257,26 @@ else if (!CollectionUtils.isEmpty(apps)) {
 				assembler);
 	}
 
+	/**
+	 * Unregisters the list of applications provided via request parameter.
+	 * @param request - Contains a list of applications to be removed
+	 * @return A response with the status of each requested application with either success or error
+	 */
+	@RequestMapping(method = RequestMethod.POST, value = "/bulk/unregister")
+	@ResponseStatus(HttpStatus.OK)
```

Comment:
maybe only 200 if all the unregister requests are successful and 207 otherwise?
  https://restpatterns.mindtouch.us/HTTP_Status_Codes/207_-_Multi-Status